themes/dannyseidel/alpenglow.json

```diff
@@ -0,0 +1,20 @@
+{  "author": "Danny Seidel",
+  "authorUsername": "DannySeidel",
+  "version": "1",
+  "name": "Alpenglow",
+  "appearance": "dark",
+  "colors": {
+    "background": "#003F71",
+    "backgroundSecondary": "#9E0000",
+    "text": "#EEEEEE",
+    "selection": "#FF7C43",
+    "loader": "#5751A1",
+    "red": "#FF1111",
+    "orange": "#F86D1C",
+    "yellow": "#FFC839",
+    "green": "#23F893",
+    "blue": "#1686F6",
+    "purple": "#7546F8",
+    "magenta": "#F847BA"
+  }
+}
```

Comment:
can you make sure this file is properly formatted as the prettier's config? if you're using VSCode you can use "Format Document" command palette

Reply:
Hi @peduarte

I fixed the formatting.
I didn't look at the formatting before as I copied the JSON directly from the Raycast Theme Studio. Maybe look into providing the JSON properly formatted to prevent that from happening to others in the future.